src/bindings/python/src/openvino/utils/node_factory.py

```diff
@@ -52,15 +52,23 @@ def create(
         if attributes is None:
             attributes = {}
 
-        assert arguments is not None
+        assert arguments is not None  # Questa asserzione aiuta Pyright a capire che arguments non è None
```

Comment:
could you please write comments in English?
OpenVINO Team is very multicultural, so we use English as a means of communication